container-make.sh

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env bash
+
+set -x
+
+CONTAINER_NAME=${1}
+CONTAINER_TAG=${2}
+
+PROJECT_NAME='github.com/tracer/tracer'
+PROJECT_DIR="${PWD}"
+
+CONTAINER_GOPATH='/go'
+CONTAINER_PROJECT_DIR="${CONTAINER_GOPATH}/src/${PROJECT_NAME}"
+CONTAINER_PROJECT_GOPATH="${CONTAINER_GOPATH}"
+
+docker run --rm \
+        -v ${PROJECT_DIR}:${CONTAINER_PROJECT_DIR} \
+        -e GOPATH=${CONTAINER_PROJECT_GOPATH} \
+        -e CGO_ENABLED=0 \
+        -e GODEBUG=netdns=go \
+        -w "${CONTAINER_PROJECT_DIR}" \
+        golang:1.7rc1-alpine \
+        go build -v -o tracer ${PROJECT_NAME}/cmd/tracer
+
+docker run --rm \
+        -v ${PROJECT_DIR}:${CONTAINER_PROJECT_DIR} \
+        -e GOPATH=${CONTAINER_PROJECT_GOPATH} \
+        -e CGO_ENABLED=0 \
+        -e GODEBUG=netdns=go \
+        -w "${CONTAINER_PROJECT_DIR}" \
+        golang:1.7rc1-alpine \
+        go build -v -o tracer-cli ${PROJECT_NAME}/cmd/tracer-cli
+
+strip "${PROJECT_DIR}/tracer"
+strip "${PROJECT_DIR}/tracer-cli"
```

Comment:
If you really really really want to strip, [I'd start here](https://blog.filippo.io/shrink-your-go-binaries-with-this-one-weird-trick/). But IMO I wouldn't bother. The handful of megabytes it might save you in your Docker image simply isn't worth the risk you introduce of weird and difficult-to-debug undefined behavior. If this were a code review I'd insist you delete these lines unless you could explicitly demonstrate a need for keeping them.